ui-tests/src/test/resources/features/customizations/monitoring-addon.feature

```diff
@@ -0,0 +1,173 @@
+# @maintainer: mmuzikar@redhat.com
+
```

Comment:
It the test ready to be run in parallel? I mean, what if the test will be run on the same cluster twice at the same time (firefox and chrome jobs).
Wouldn't it be better to move the test to the rest package? DB, datamapper, and timer steps are already implemented there so you just copy-paste them here and replace UI equivalent)